core/warning.rbs

```diff
@@ -29,6 +29,7 @@
 # The `warning` gem provides convenient ways to customize Warning.warn.
 #
 module Warning
+  type category = :deprecated | :experimental | :performance
```

Comment:
Do we have `:performance` category?

```ruby
3.2.2 :002 > Warning[:performance]
(irb):2:in `[]': unknown category: performance (ArgumentError)
	from (irb):2:in `<main>'
	from /Users/soutaro/.rvm/gems/ruby-3.2.2/gems/irb-1.7.4/exe/irb:9:in `<top (required)>'
	from /Users/soutaro/.rvm/gems/ruby-3.2.2/bin/irb:25:in `load'
	from /Users/soutaro/.rvm/gems/ruby-3.2.2/bin/irb:25:in `<main>'
```

Reply:
In 3.3.0 we do, it was just added

Reply:
Hmm. RBS definition is based on the latest release of Ruby, 3.2 now.
So..., could you delete it? (because test is failing anyway.)

Reply:
Sure, I'll do that with a few other things I've added from 3.3